src/reducers/dialogReducer.ts

```diff
@@ -6,7 +6,10 @@ const initialState = {
   type: null,
   properties: {},
   x: null,
-  y: null
+  y: null,
+  isNodeDialog : false,
+  edgeFrom : null,
+  edgeTo: null

```

Comment:
`type` and `isNodeDialog` seem redundant. You should just set `type` to node or edge. That way you can continue to expand the different types of dialogs.